### DISCUSSION TITLE
Three finger swipe, page flip on drawings and side to side

### DISCUSSION THREAD

**Discussion by miya-sans** [keiwando/toonsquid#291] [Ideas]
I come from a traditional animation background, and have yet come to a better solution to flipping through frames then the three finger swipe gesture.
I have some suggestions:
Allow flipping between drawings rather frames, similar to the option for the forward/backward buttons.
In traditional animation it is common to only flip between two drawings in rapid succession. If all your images are one frame, this won't be a problem, but if you already set timing beyond one frame then sometimes you'll just be flipping through the same drawings multiple times.
Secondly, try allowing side to side three finger gesture to flip through frames. Three finger up and down is an asymmetrical action in that it's much easier to do downwards than upwards.
When in-betweening there is a repetitive motion of rapidly flipping between three or more frames that the up down motion has a low accuracy of detection, and replication.
In addition, as it appears there's no way to edit the color of vector strokes, having an option to tint a vector layer to a specific color like in blender greasepencil, would help tremendously for organizing multiple drawing layers within a layer. 
Finally, a bit unrelated, but perspective and warp transform in vector mode, this establishes similar functionality between raster and vector.
Overall, I must say that ToonSquid is one of the best animation apps I've used, it has great gestures from iPad apps, and the animation functionality of a desktop application. Great work!

**Comment by Nishrodrigo** [comment]
When using keyboard shortcuts for flipping between drawings is a must have for me, so i dont need to press insanely the < > buttons :)

**Reply by diegostoliar** [reply]
I feel that there should be a button that you press and while you are holding it moving your pen left and right flips the frames or drawings (optionally), and there should be a ui button and on a keyboard shortcut for it.

This way your pen AND your eyes never leave the drawing, which would speed animation tweaking quite a bit.

**Reply by Nishrodrigo** [reply]
Love that idea! For me just a simple keyboard shortcut is enough, my thinking is the arrow keys, when transform is not  active the arrow keys flips the drawings and < > flip the frames like in rough animator app
